reactapp/package.json

```diff
@@ -13,32 +13,33 @@
   "private": true,
   "proxy": "https://demo.hyva.io",
   "dependencies": {
-    "@heroicons/react": "^1.0.3",
+    "@heroicons/react": "^1.0.1",
+    "@hyva/react-checkout": "file:../../../../../vendor/hyva-themes/magento2-hyva-checkout/src/reactapp/src",
```

Comment:
Not sure why this is here when the `aliasMap` setting is used.

Reply:
This will copy the original React components inside `node_modules` directory. The advantage of that is our IDE is now able to recognize the file when we refer to it with `@hyva/react-checkout` prefix. So no more complaints + auto-complete feature would work perfectly again.